src/context.rs

```diff
@@ -224,15 +224,17 @@ pub fn get_json_pointer(key: &str) -> String {
         // to fix https://github.com/Keats/tera/issues/590
         static ref JSON_POINTER_REGEX: regex::Regex = regex::Regex::new(r#""[^"]*"|[^.]+"#).unwrap();
     }
-
+    let mut res = String::with_capacity(key.len() + 1);
```

Comment:
Would doing `key.len() * 2` be better to make sure no reallocation happens?

Reply:
we know the exact size will be max key.len() +1
In case of the map it could be less, not more.

Reply:
Yeah nevermind I realised I was wrong after posting the comment